pymodbus/repl/server/cli.py

```diff
@@ -80,7 +81,9 @@
 
 
 def info(message):
-    click.secho(str(message), fg="green")
+    if not isinstance(message, str):
```

Comment:
I understand most of the changes, but this one seems unrelated.

Reply:
Not related and does not affect the normal clients. This is only for REPL client.